### PR TITLE
Fixing details

### PR DESCRIPTION
**Pull request by yceruto**
* Fixing imports (unused or missing).
* Adding type hinting.
* Fixed variable definition.

### CODE REVIEW

File: src/AppBundle/Command/DeleteUserCommand.php [hunk 1]
```diff
@@ -109,6 +109,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->usernameValidator($username);
 
         $repository = $this->entityManager->getRepository('AppBundle:User');
+        /** @var User $user */
```
Comment (voronkovich) [inline src/AppBundle/Command/DeleteUserCommand.php]: @yceruto, I think it's better to use a FQCN here.

Comment (yceruto) [inline src/AppBundle/Command/DeleteUserCommand.php]: @voronkovich why it's better to use a FQCN here? please.

Comment (voronkovich) [inline src/AppBundle/Command/DeleteUserCommand.php]: @yceruto, using FQCN doesn't require to import User class.

Comment (yceruto) [inline src/AppBundle/Command/DeleteUserCommand.php]: Ok, fixed! thanks @voronkovich

File: src/AppBundle/Command/DeleteUserCommand.php [hunk 2]
```diff
@@ -109,6 +108,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->usernameValidator($username);
 
         $repository = $this->entityManager->getRepository('AppBundle:User');
+        /** @var \AppBundle\Entity\User $user */
```
Comment (xabbuh) [inline src/AppBundle/Command/DeleteUserCommand.php]: I would use a `use` statement for this class (that's what we do in the core).

Comment (voronkovich) [inline src/AppBundle/Command/DeleteUserCommand.php]: @xabbuh, what the advantages of importing a class that we don't really use?

File: src/AppBundle/Command/DeleteUserCommand.php (hunk 1 above)
Comment (yceruto) [inline src/AppBundle/Command/DeleteUserCommand.php]: Changed by @xabbuh comment.

**Comment by javiereguiluz** [comment]
Thanks @yceruto.

**Pull request closed without merging**